ui/components/app/flask/snaps-authorship-pill/snaps-authorship-pill.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Chip from '../../../ui/chip';
+import Box from '../../../ui/box';
+import Typography from '../../../ui/typography';
+import {
+  COLORS,
+  TYPOGRAPHY,
+} from '../../../../helpers/constants/design-system';
+
+const SnapsAuthorshipPill = ({ packageName, className, url }) => {
+  return (
+    <a href={url}>
```

Comment:
Do we want this to open in a new tab? Or should we allow `...props` to be passed to the root element to give that option?
```suggestion
    <a href={url} target="_blank" rel="noopener noreferrer">
```

Reply:
I think it's safe to just allow opening in a new tab, I'll fix this